test/studies/shootout/meteor/kbrady/meteor-parallel-alt.chpl

```diff
@@ -358,8 +352,8 @@ proc searchLinearHelper(in board, in pos, in used, in placed,
 //
 record BackoffSpinLock {
   var l: atomic bool,
-      lockAttempts: uint,
-      maxLockAttempts = (2**16-1): uint;
+      lockAttempts = 0,
+      maxLockAttempts = (2**16-1);
```

Comment:
I believe these 2 need to be uint still, for the additional upper bound. @ronawho can confirm this. ( I originally suggested a similar change)